libs/web/test-environment/src/lib/edit/test-environment-edit.component.ts

```diff
@@ -113,8 +113,10 @@ export class TestEnvironmentEditComponent {
 
     const environmentDto: TestEnvironment = {
```

Comment:
Why is there a name which is not setted after update?

Reply:
Fixed in e212d0a.